### PR TITLE
plot_line() to plot a small sphere if a single coordinate is passed

### PR DESCRIPTION
**Pull request by jeffreypaul15**
- `plot_line()` is changed to `plot_coordinates()`
- If the user passes a single coordinate, the plot produced is a small sphere (0.05 times `R_sun`)

image

- [ ] Tests are to be added
- [ ] Example to be re-written to plot on the brightest coordinate?

### CODE REVIEW

**Comment by nabobalis** [comment]
> * [ ]  Example to be re-written to plot on the brightest coordinate?

Why not? Could be nice but would we want a sphere to cover the part fully?

**Comment by jeffreypaul15** [comment]
> Why not? Could be nice but would we want a sphere to cover the part fully?

Thought it would be consistent with sunpy's examples but there's no need to I suppose

**Comment by nabobalis** [comment]
> > Why not? Could be nice but would we want a sphere to cover the part fully?
> 
> Thought it would be consistent with sunpy's examples but there's no need to I suppose

Lets do it.

**Review by dstansby — CHANGES_REQUESTED** [review]
Looks good, but the test just needs fixing

**Comment by jeffreypaul15** [comment]
> Looks good, but the test just needs fixing

I'm sorry I don't understand what has to be fixed

**Comment by nabobalis** [comment]
> > Looks good, but the test just needs fixing
> 
> I'm sorry I don't understand what has to be fixed

```
>       assert plotter.plotter.mesh.n_cells == 1
E       assert 1680 == 1
```

File: changelog/29.feature.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+``plot_line()`` is renamed to :meth:`~sunkit_pyvista.plotter.SunpyPlotter.plot_coordinates`
+which also allows for a sphere with given ``radius`` to be plotted if a sinlge coordinate is passed to it.
```
Comment (nabobalis) [inline changelog/29.feature.rst:2]:
```suggestion
which also allows for a sphere with given ``radius`` to be plotted if a singe coordinate is passed to it.
```

File: sunkit_pyvista/plotter.py [hunk 2]
```diff
@@ -134,23 +134,33 @@ def plot_map(self, m, **kwargs):
...
-        Plot a line from a set of coordinates.
+        Plot a sphere if a single coordinate is passed and
+        plots a line if multiple coordinates are passed.
 
         Parameters
         ----------
         coords : `astropy.coordinates.SkyCoord`
-            Coordinates to plot as a line.
+            Coordinate(s) to plot as a center of sphere or line.
+        radius : `int`
```
Comment (nabobalis) [inline sunkit_pyvista/plotter.py:146]:
```suggestion
        radius : `int`, optional
```

File: sunkit_pyvista/plotter.py [hunk 3]
```diff
@@ -134,23 +134,33 @@ def plot_map(self, m, **kwargs):
...
+            Radius of the sphere times the radius of the sun
+            to be plotted when a single coordinate is passed.
+            Defaults to ``0.05`` times the radius of the sun.
         **kwargs :
             Keyword arguments are passed to `pyvista.Plotter.add_mesh`.
         Notes
```
Comment (nabobalis) [inline sunkit_pyvista/plotter.py:152]:
```suggestion

        Notes
```

**Review by dstansby — APPROVED** [review]

**Pull request merged**